searches/linear_search.py

```diff
@@ -38,6 +38,41 @@ def linear_search(sequence, target):
     return None
 
 
+def rec_linear_search(sequence: list, low: int, high: int, target: int) -> int:
+    '''
+    Pure implementation of recursive linear search algorithm in Python
+
+    :param sequence: An array of items
+    :param low: Lower bound of the array
+    :param high: Higher bound of the array
+    :param target: The element to be found
+    :return: Index of the key or -1 if key not found or None in case of an exception
+
+    Examples:
+    >>> rec_linear_search([0, 30, 500, 100, 700], 0, 4, 0)
+    The element 0 is present at index 0
```

Comment:
This function returns an int.
```suggestion
    0
```